packages/shared/utils/index.ts

```diff
@@ -104,20 +101,3 @@ export function objectPick<O, T extends keyof O>(obj: O, keys: T[], omitUndefine
     return n
   }, {} as Pick<O, T>)
 }
-
-/**
- * Simple recursive deep clone
- *
- * @category Object
- */
-export const cloneDeep = (obj: any) => {
```

Comment:
Please revert this change

Reply:
you wanna to use this as default function instead `JSON.parse(JSON.stringfy(...))`?